tests/components/hand-controls.test.js

```diff
@@ -0,0 +1,153 @@
+/* global assert, process, setup, suite, test */
+var entityFactory = require('../helpers').entityFactory;
+var controllerComponentName = 'hand-controls';
+
+suite(controllerComponentName, function () {
+  setup(function (done) {
+    var el = this.el = entityFactory();
+    el.setAttribute(controllerComponentName, '');
+    process.nextTick(function () {
```

Comment:
What is the assertion that is exactly failing without `nextTick`? Component updates are synchronous since AFrame 0.2.0. If not there might be a bug somewhere. There might be still some remaining tests that assume the old asynchronous situation.